docker-compose.yml

```diff
@@ -103,7 +103,7 @@ services:
       - xpack.security.enabled=false
       - ES_JAVA_OPTS=-Xms512m -Xmx512m
     ports:
-      - "9200:9200"
+      - "9201:9200"
```

Comment:
IIUC, this means port 9201 inside the container, but localhost:9200 from the outside?

I see that port 9200 appears in kuma/settings/common.py and kuma/search/tests/test_views_admin.py  If neither of those need to change, then this looks good to me.

Reply:
It's actually the opposite (see https://docs.docker.com/compose/compose-file/compose-file-v2/#ports). The first number is the port number on the host (so outside the docker-compose environment), and the second is the port number for that container inside the docker-compose environment.